NDTensors/src/lib/Sectors/test/test_category.jl

```diff
@@ -157,13 +125,8 @@ using Test: @test, @testset
     @test dual(ı) == ı
     @test dual(τ) == τ
 
-    @test dimension(ı) == 1
+    @test dimension(ı) === 1.0
```

Comment:
I think we don't need to be so restrictive with this test, I prefer the older version.

I think really what you want is to test that `dimension` is type stable, which you can do with:
```julia
@inferred dimension(ı)
```